### ISSUE TITLE
Translation File Select

### ISSUE THREAD

**Issue by aersamkull** [Roman-Lo/vscode-ng-i18n-editor#4]
Hi there
Thanks for this great extension! 
One thing I could not get to work is to specify the location of the target file. The default convention is not the same as xliffmerge is using which is problematic to me. Is it possible to just choose 2 xliff files instead of providing a file and a target locale?

**Comment by Roman-Lo** [comment]
I read the `ngx-i18nsupport` documentation for `xliffmerge` command slightly. So, if I make the editor support editing for single xliff, will this solve your problem?

**Comment by aersamkull** [comment]
I think it will. My use case is actually simple: I have a messages.de.xlf I want to translate meaning add target tags in this file)

**Comment by Roman-Lo** [comment]
> I think it will. My use case is actually simple: I have a messages.de.xlf I want to translate meaning add target tags in this file)

I've published a new version (0.2.0). In order to gain more flexibility, I added some new settings. By your case, you can update your `.ngI18nconfig` setting by the following:
```jsonc
{
  locales: ['de-DE'], // add all your translation locales in this field.
  editor: {
     "messageLocations": [
        "<rel/path/from/workspace/to/your/source/message.xlf>"
      ],
     "mode": "target-file", // 'target-file' mode allows you to edit directly on the target xliff (based on the `messageLocations` and the given `translationFileNamePattern`)
     "translationFileNamePattern": "${name}.{lang}" // using this field to resolve your translation xliff location
  }
}
```
There is one more new setting `emptyTranslationHandling`, which can let you to control how to deal with the empty translation, since I saw `xliffmerge` allows you to reserve the 'useless' trans unit, and also give you the source message as the default translation value for the trans unit.

I've updated the README.md, so you can find all the available configuration settings in it. 

I'm closing this issue. But if you still have questions, please feel free to leave a message. Hope you like this extension.
